roles/openshift_openstack/tasks/check-prerequisites.yml

```diff
@@ -17,6 +17,23 @@
     that: 'shade_result.rc == 0'
     msg: "Python module shade is not installed"
 
+# Gather Neutron extension facts
+- name: Check for Neutron trunk support
+  os_network_extension_facts:
```

Comment:
This must be called exactly as the library python filename, i.e. `os_network_extensions`.

Reply:
Done